src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -480,6 +497,26 @@ internal static bool VersionLessThanOrEquals(string a, string b)
             return SimpleVersion.Parse(a) <= SimpleVersion.Parse(b);
         }
 
+        private static object ParseNuGetFramework(string tfm)
```

Comment:
Putting these here exposes them as `[MSBuild]::` functions. That's what a lot of our stuff does. Should we consider making it more like `[NuGet]::`? That's a bigger change for not a ton of benefit. I think I lean toward "nah".

Reply:
All these are dotnet related, not just NuGet specific, so why not under `[DotNet]::`. It is easy to understand that way!

Putting it under MSBuild, atleast for me, is not clear, I read `IsCompatibleWith` and I'm confused as to, for what part of the tech stack is it comparing to, is it .NET? VC? PHP? etc...